AI/Nullkiller/Pathfinding/AINodeStorage.cpp

```diff
@@ -1203,7 +1203,7 @@ void AINodeStorage::calculateTownPortalTeleportations(std::vector<CGPathNode *>
 	std::vector<const ChainActor *> actorsVector(actorsOfInitial.begin(), actorsOfInitial.end());
 	tbb::concurrent_vector<CGPathNode *> output;
 
-	if(actorsVector.size() * initialNodes.size() > 1000)
+	if(false) //if (actorsVector.size() * initialNodes.size() > 1000)
```

Comment:
is this only for a test?

Reply:
Not sure. If I can figure out what causes thread race & how to fix it - yes.
If not - then will keep as it until @nullkiller can look into it. Slight performance loss is better than crash.

Reply:
then it'd be better to use `if 0` and leave a TODO